cmd/restic/cmd_generate.go

```diff
@@ -63,22 +65,30 @@ func writeManpages(dir string) error {
 }
 
 func writeBashCompletion(file string) error {
-	Verbosef("writing bash completion file to %v\n", file)
+	if terminal.IsTerminal(int(os.Stdout.Fd())) {
```

Comment:
Please use `stdoutIsTerminal()`. There's no need to copy paste the terminal detection code everywhere.

Reply:
Oh, I didn't see that function. I was even tempted now to put there another layer of abstraction as in
```
func openOutput(file string, format string, args ...any) (io.WriteCloser, error) {
  if file == "-" {
    return os.Stdout, nil
  }
  if stdoutIsTerminal() {
    Verbosef(format, args...)
  }
  return os.Create(file)
 
 ```
 And then use `cmdRoot.Gen*Completion(w)`